`yarn run docs` does not work anymore

## 📝 Provide detailed reproduction steps (if any)

Execute `yarn run docs` (full setup needed).

### ✔️ Expected result

No errors.

### ❌ Actual result

```
ERROR in ./node_modules/axios/lib/utils.js 4:0-28
Module not found: Error: Can't resolve 'crypto' in '/home/pomek/Projects/ckeditor/ckeditor5/node_modules/axios/lib'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "crypto": require.resolve("crypto-browserify") }'
        - install 'crypto-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "crypto": false }
resolve 'crypto' in '/home/pomek/Projects/ckeditor/ckeditor5/node_modules/axios/lib'
```

### ❓ Possible solution

* Downgrade `axios` to the latest working version.

## 📃 Other details

* Two hours ago, there was a new `axios` release: https://www.npmjs.com/package/axios/v/1.8.0

![Image](https://github.com/user-attachments/assets/65b4433c-d0bd-4ff8-b202-cccfbb8f1d47)

--- a/scripts/docs/snippetadapter.mjs
+++ b/scripts/docs/snippetadapter.mjs
@@ -478,6 +478,9 @@ function getWebpackConfig( snippets, config ) {
 			extensions: [ '.ts', '.js', '.json' ],
 			extensionAlias: {
 				'.js': [ '.js', '.ts' ]
+			},
+			fallback: {
+				crypto: false
 			}
 		},
 

